### ISSUE TITLE
save/load: support import/export of image tarballs

### ISSUE THREAD

**Issue by sureshsankaran** [rancher/k3c#27]
I didn't see supported options of k3s to include docker save/load equivalent. Could you please add it, as it is life saver when the device is in offline mode and not having to setup private registry?

**Comment by dweomer** [comment]
@sureshsankaran yes, we want to support this. The first cut of this will likely copy import/export from `ctr` but I think the tarball format is slightly incompatible with Docker.
